### ISSUE TITLE
[Card-related]: Cannot open link in React Wrapper mobile Safari VoiceOver

### ISSUE THREAD

**Issue by RichKummer** [carbon-design-system/carbon-for-ibm-dotcom#8328]
### Description

Multiple Card-related components that do not have text CTAs do not navigate user on double tap. It looks like if the CTA has text it works fine. If there is heading or body, those will be highlighted, but there is no way to use the link. Card static and others with text link can have the link selected to navigate to the new page.
![IMG_2036](https://user-images.githubusercontent.com/15643582/154507360-915641fe-f69f-47a9-970d-21fa1d3c2fe4.PNG)

![IMG_2035](https://user-images.githubusercontent.com/15643582/154507356-31bfa92c-d8d4-41f3-bab0-8c88e27090ad.PNG)

### Component(s) impacted

- Card in card
- Card link
- Card logo
- Carousel
- Card group
- Content block simple with Card CTA
- Feature section

### Browser

Safari

### Carbon for IBM.com version

v1.31.0-rc.1

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

C4IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components/examples/codesandbox/components-react/card

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by IgnacioBecerra** [comment]
@RichKummer Seems like this issue has already been discussed in #8581, where this behavior is the intended one for screen readers. 

Leaning towards closing this issue, but would like to hear your thoughts on it first.

**Comment by kennylam** [comment]
I agree, this is the same issue as #8581 and can be closed.
